dcicutils/deployment_utils.py

```diff
@@ -58,7 +64,282 @@ def boolean_setting(settings, key, default=None):
         return setting
 
 
-class Deployer:
+class EBDeployer:
+
+    S3_BUCKET = 'dcic-application-versions'
+    EB_APPLICATION = '4dn-web'  # XXX: will need to change if this changes -Will
+    EB_CONFIGURATION_SETTINGS = 'ConfigurationSettings'
+    EB_OPTION_SETTINGS = 'OptionSettings'
+    EB_ENV_VARIABLE_NAMESPACE = 'aws:elasticbeanstalk:application:environment'
+    DEFAULT_INDEXER_SIZE = 'c5.4xlarge'
+    UPDATING = 'Updating'
+    STATUS = 'Status'
```

Comment:
If these were at toplevel instead of in the class, you'd not have to refer to them with `cls.` prefixes. In terms of modularity, that's a little less connected, but really you're not relying on these names for their class-oriented definition and also you don't want anyone setting the value of these in the class.